### PR TITLE
Enabling the ImageFiltered(ImageFilter.matrix) page of macrobenchmark

### PR DESCRIPTION
**Pull request by flar**
This change enables the ImageFiltered benchmark variant of the existing filtered child benchmark page in preparation for measuring the impact of some upcoming performance enhancements targeted at that widget.

### CODE REVIEW

**Comment by flar** [comment]
Thoughts on the use of the page name identifiers (specifically `kScrollableName`) directly in util.dart? The assumption is that the macroPerfTest() function is specific to these tabbed macrobenchmarks.

**Review by liyuqian — APPROVED** [review]
The use of `kScrollableName` looks good to me.

File: dev/benchmarks/macrobenchmarks/lib/common.dart [hunk 1]
```diff
@@ -13,3 +13,6 @@ const String kTextRouteName = '/text';
 const String kAnimatedPlaceholderRouteName = '/animated_placeholder';
 const String kColorFilterAndFadeRouteName = '/color_filter_and_fade';
 const String kFadingChildAnimationRouteName = '/fading_child_animation';
+const String kImageFilteredTransformAnimationRouteName = '/imagefiltered_transform_animation';
+
+const String kScrollableName = "/macrobenchmark_listview";
```
Comment (liyuqian) [inline dev/benchmarks/macrobenchmarks/lib/common.dart:18]: format issue: single quote instead of double quote

Comment (flar) [reply dev/benchmarks/macrobenchmarks/lib/common.dart]: Done.

**Pull request merged**